PyInstaller/depend/bindepend.py

```diff
@@ -20,6 +20,7 @@
 # Required for extracting eggs.
 import zipfile
 from glob import glob
+from subprocess import run, PIPE
```

Comment:
Can I be picky and request that instead of importing symbols from the module, we instead import the module and refer to the symbols using their fully-qualified name? (i.e, `import subprocess` and `subprocess.run` and `subprocess.PIPE`)?

Reply:
[Here](https://github.com/pyinstaller/pyinstaller/pull/6245/commits/fb191126ce11f7d244a7a3624f1fafd2d25f527c#diff-449dff4a40028675f338ef14a95fad67edea219ff61fdb70940b7e62f31ac329R95-R97) too? Or do you not mind for locally scoped imports?

Reply:
I find locally scoped imports less problematic, because it's more apparent where that `run` came from. But overall I prefer fully-qualified names for standard library stuff...